spec/index.html

```diff
@@ -306,48 +308,48 @@ <h2>Security Considerations</h2>
     be obfuscated due to presentation of such characters.</p>
 
   <p>N-Quads is a general-purpose assertion language;
-    applications may evaluate given data to infer more assertions or to dereference IRIs,
+    applications may evaluate given data to infer more assertions or to dereference <a>IRIs</a>,
     invoking the security considerations of the scheme for that IRI.
-    Note in particular, the privacy issues in [[!RFC3023]] section 10 for HTTP IRIs.
+    Note in particular, the privacy issues in [[RFC3023]] section 10 for HTTP IRIs.
     Data obtained from an inaccurate or malicious data source may lead to inaccurate or misleading conclusions,
     as well as the dereferencing of unintended IRIs.
     Care must be taken to align the trust in consulted resources with the sensitivity of
     the intended use of the data;
     inferences of potential medical treatments would likely require different trust than inferences
     for trip planning.</p>
 
-  <p>N-Quads is used to express arbitrary application data; security considerations
-    will vary by domain of use. Security tools and protocols applicable to text
+  <p>N-Quads is used to express arbitrary application data;
```

Comment:
```suggestion
  <p>N-Quads serialization is used to express arbitrary application data;
```

Reply:
I don't think that really scans either; could be "An N-Quads serialization ...",, but that breaks the general form. Generally, I would probably say that "N-Quads _is_ used to express ..." although technically it is a specific use of N-Quads that accomplishes that. In this case, we're discussing how the language is used.

Reply:
Perhaps, "The N-Quads language is used..."?

Reply:
```suggestion
  <p>The N-Quads language is used to express arbitrary application data;
```